### PR TITLE
readme: Add reference to pasta in the readme

### PR DESCRIPTION
**Pull request by it-a-me**
Fixes https://github.com/containers/podman/issues/24244

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Podman uses OCI projects and best of breed libraries for different aspects:
 - Runtime: We use the [OCI runtime tools](https://github.com/opencontainers/runtime-tools) to generate OCI runtime configurations that can be used with any OCI-compliant runtime, like [crun](https://github.com/containers/crun/) and [runc](https://github.com/opencontainers/runc/).
 - Images: Image management uses the [containers/image](https://github.com/containers/image) library.
 - Storage: Container and image storage is managed by [containers/storage](https://github.com/containers/storage).
-- Networking: Networking support through use of [Netavark](https://github.com/containers/netavark) and [Aardvark](https://github.com/containers/aardvark-dns).  Rootless networking is handled via [slirp4netns](https://github.com/rootless-containers/slirp4netns).
+- Networking: Networking support through use of [Netavark](https://github.com/containers/netavark) and [Aardvark](https://github.com/containers/aardvark-dns).  Rootless networking is handled via [pasta](https://passt.top/passt) and [slirp4netns](https://github.com/rootless-containers/slirp4netns).
```
Comment (Luap99) [inline README.md:94]: this should likely say pasta *or* slirp4netns and NOT *and* because on of them is enough

**Comment by mheon** [comment]
LGTM once comment from @Luap99 is addressed

File: README.md (hunk 1 above)
Comment (Luap99) [reply README.md:94]: @it-a-me this is still saying `and` can you update this to `or` please? thanks

Comment (it-a-me) [reply README.md]: Should be fixed now.  Sorry, I forgot to push it

**Review by Luap99 — APPROVED** [review]
/lgtm

Thanks

**Pull request merged**